src/js/site/nav.js

```diff
@@ -52,22 +52,53 @@ $(document).ready(function () {
   });
 })();
 
-// Open dropdown in language selector
+/**
+ * openLangDropdown handles setting up the toggling of our language picker
+ * on mobile and on desktop.
+ * */
 const openLangDropdown = () => {
-  const globeBtn = $('#globe-lang-btn');
-  const langPicker = $('#lang-picker');
+  const globeBtn = document.getElementById('globe-lang-btn');
+  const langPicker = document.getElementById('lang-picker');
+  const underlyingMobileMenu = document.querySelector('.mobile-sidebar');
 
-  $('body').on('mouseup', (e) => {
-    globeBtn.on('click', () => {
-      langPicker.toggleClass('lang-active');
-    });
+  // These vars are specific to the mobile picker.
+  const dimmer = document.querySelector('.lang-picker-mobile-dimmer');
+  const mobileBackBtn = document.getElementById('lang-picker-back-button');
+  const mobileCloseBtn = document.getElementById('lang-picker-close-button');
+  const globalNav = document.querySelector('.global-nav--nav');
+
+  // open the menu
+  globeBtn.addEventListener('click', () => {
+    langPicker.classList.toggle('lang-active');
+    underlyingMobileMenu.classList.add('hidden-md');
+    dimmer.style.display = 'inherit';
```

Comment:
This is more portable:

```suggestion
    dimmer.css('display', 'inherit');
```

Reply:
`.css()` is a jquery method and not applicable on vanilla js.

Reply:
we got to stop mixing vanilla and jquery all the time 🙈